stack/run/run.Dockerfile

```diff
@@ -17,6 +17,8 @@ RUN mkdir -p /tiny/tmp /tiny/var/lib/dpkg/status.d/
 RUN mkdir -p /tiny/home/nonroot \
     && chown 65532:65532 /tiny/home/nonroot
 
+RUN echo "Package: $packages\nPin: release c=multiverse\nPin-Priority: -1\n\nPackage: $packages\nPin: release c=restricted\nPin-Priority: -1\n" > /etc/apt/preferences
```

Comment:
I assume that the reason we don't `rm /etc/apt/preferences` in the tiny stack is because that filesystem is never transferred to the actual resulting filesystem (i.e. we `COPY --from-builder /tiny/` and ignore `/etc/`)?

Reply:
> I assume that the reason we don't `rm /etc/apt/preferences` in the tiny stack is because that filesystem is never transferred to the actual resulting filesystem (i.e. we `COPY --from-builder /tiny/` and ignore `/etc/`)?

Yes, I skipped removing that b/c it's just the `build` container and won't affect the final image. That's only possible on `tiny`, I assume.

Reply:
Yes, Tiny is the only stack that Paketo builds with this method, so in the `run` image for other stacks we would explicitly `rm /etc/apt/preferences` like we do for the `build` image here.